lib/parser/objects.ts

```diff
@@ -203,30 +203,40 @@ export class ObjectBase {
 export abstract class OGeneric extends ObjectBase implements I.IHasDefinitions, I.IHasReferenceLinks {
   parent: OEntity;
   definitions: (OGeneric | OPackage)[] = [];
-  referenceLinks: OReference[] = [];
+  referenceLinks: OName[] = [];
   aliasReferences: OAlias[] = [];
   lexerToken: OLexerToken;
 }
 export class OGenericConstant extends OGeneric implements I.IVariableBase, I.IHasTypeReference, I.IHasDefaultValue {
   definitions: OGenericConstant[] = [];
-  typeReference: OReference[] = [];
-  defaultValue?: OReference[] = [];
+  typeReference: OName[] = [];
+  defaultValue?: OName[] = [];
 
 }
-export class OReference extends ObjectBase implements I.IHasDefinitions, I.IHasReferenceToken {
+export class OName extends ObjectBase implements I.IHasDefinitions, I.IHasReferenceToken {
   definitions: ObjectBase[] = [];
   notDeclaredHint?: string;
   lexerToken: undefined;
-  constructor(public parent: ObjectBase, public referenceToken: OLexerToken, range?: OIRange) {
+  // Workaround for checking of OWrites in associations. Because of overloading they can not be correctly checked.
+  // This avoid false positives
```

Comment:
```suggestion
  // This avoids false positives
```
which kind of false positives?